packages/accounts-base/accounts_server.js

```diff
@@ -176,9 +176,10 @@ Ap._failedLogin = function (connection, attempt) {
   });
 };
 
-Ap._successfulLogout = function () {
+Ap._successfulLogout = function (connection, userId) {
+  const user = userId && this.users.findOne(userId);
```

Comment:
onLogin does get an object with user property. See following code. This is also mentioned in the documentation and we indeed use this property in our project. 

  var user;
  if (result.userId)
    user = this.users.findOne(result.userId);

  var attempt = {
    type: result.type || "unknown",
    allowed: !! (result.userId && !result.error),
    methodName: methodName,
    methodArguments: _.toArray(methodArgs)
  };
  if (result.error)
    attempt.error = result.error;
  if (user)
    attempt.user = user;